servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/AbstractStreamingHttpConnection.java

```diff
@@ -102,11 +103,15 @@ public final HttpConnectionContext connectionContext() {
                 this::newSplicedResponse));
     }
 
+    private static boolean shouldOffload() {
```

Comment:
can we leverage `BooleanSupplier shouldOffload(IoExecutor executor)` here?

Reply:
yes